applications_tests/dem_2d/circle_restart.output

```diff
@@ -10,23 +10,23 @@ Finished reading triangulation
 10 particles of type 0 were inserted, 0 particles of type 0 remaining
 ***************************************************************** 
 id, type, dp, x, y, z 
-0 0 0.00500 0.0034 -0.0972
-1 0 0.00500 0.0005 -0.0928
-2 0 0.00500 -0.0115 -0.0964
-3 0 0.00500 0.0084 -0.0967
-4 0 0.00500 0.0183 -0.0957
-5 0 0.00500 -0.0262 -0.0935
-6 0 0.00500 -0.0016 -0.0973
-7 0 0.00500 -0.0065 -0.0969
-8 0 0.00500 -0.0165 -0.0959
-9 0 0.00500 0.0134 -0.0962
-10 0 0.00500 -0.0214 -0.0949
-11 0 0.00500 -0.0131 -0.0916
-12 0 0.00500 -0.0044 -0.0923
-13 0 0.00500 0.0055 -0.0926
-14 0 0.00500 0.0105 -0.0921
-15 0 0.00500 0.0185 -0.0907
-16 0 0.00500 0.0231 -0.0944
-17 0 0.00500 0.0279 -0.0929
-18 0 0.00500 0.0327 -0.0915
-19 0 0.00500 -0.0181 -0.0911
+0 0 0.00500 -0.0257 -0.0936
```

Comment:
I didn't save the parameter handler files that I had used to create the restart files. I tried several parameter handler files, but couldn't recreate the same information. To avoid having this problem next time, I have saved these parameter handler files in Lethe-utils.